ansible-dryad/roles/dryad_app/templates/bash_profile.j2

```diff
@@ -10,6 +10,7 @@ export PGHOST={{ dryad.db.host }}
 export PGPORT={{ dryad.db.port }}
 export PGUSER={{ dryad.db.user }}
 export PGDATABASE={{ dryad.db.name }}
+export PGPASSWORD={{ dryad.db.password }}
```

Comment:
The .pgpass appears to be working fine now. If we set this, the .pgpass will essentially be ignored.

Reply:
You were complaining before about not having it set for import_db_dump, so I thought it'd be better to have it set than not.